pam/internal/adapter/authmodeselection.go

```diff
@@ -31,6 +31,9 @@ type supportedUILayoutsReceived struct {
 	layouts []*authd.UILayout
 }
 
+// SupportedUILayoutsSet is the event signalling that the current supported ui layout in the context have been set.
+type SupportedUILayoutsSet struct{}
```

Comment:
It seems this one can be private?

Reply:
yeah, I was keeping it non-private since it's something actually consumed by `brokerselection` so I kept it "public"... 
I feel we should cleanup all them though, since we've a bit of mixture of private and public definitions for those messages.

Reply:
Changed though...